impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -311,6 +319,21 @@ std::optional<Entity> GaussianBlurFilterContents::RenderFilter(
   Vector2 pass1_pixel_size =
       1.0 / Vector2(pass1_out.value().GetRenderTargetTexture()->GetSize());
 
+  Quad blur_uvs = {Point(0, 0), Point(1, 0), Point(0, 1), Point(1, 1)};
+  if (expanded_coverage_hint.has_value() &&
+      input_snapshot_coverage.has_value()) {
+    std::optional<Rect> uvs = MakeReferenceUVs(input_snapshot_coverage.value(),
```

Comment:
It has to be optional because technically the result of `Intersection` could be nothing if there is no intersection.  That would be indicative of some larger bug.  Drawing the whole texture should never be wrong, just suboptimal.  I added a DCHECK.